video/out/hwdec/dmabuf_interop_gl.c

```diff
@@ -186,6 +186,7 @@ static bool vaapi_gl_map(struct ra_hwdec_mapper *mapper,
                 format[2] = DRM_FORMAT_R8;
                 break;
             case DRM_FORMAT_P010:
+            case DRM_FORMAT_P030:
```

Comment:
This needs to be conditionalised. It was added in a newer libdrm than our minimum version.

Reply:
I'm not really sure how to do this. Would it require use of ioctl? I see stuff like LIBASS_VERSION, is there an equivalent here?

Reply:
Maybe `#ifdef DRM_FORMAT_P030`? I'm assuming @philipl is referring to the library version, not the kernel's implementation of DRM.

Reply:
Yes indeed. It's just a #define, so an #ifdef check is all that's required.